okapi/core.php

```diff
@@ -28,7 +28,7 @@
 function get_admin_emails()
 {
     $emails = array();
-    if (class_exists("okapi\\Settings"))
+    if (class_exists(Settings::class))
```

Comment:
i checked it with the following snippet:

```php
<?php
namespace okapi;

var_dump(class_exists(Settings::class));
```
via https://3v4l.org
It works on all php versions